definitions/apm-application/relationships.yml

```diff
@@ -0,0 +1,26 @@
+relationships:
+  # Select entityGuid, peer.hostname from Span where peer.hostname is not null and category =
+  # 'datastore'
+  # ExtService in this case is a DB
+  - instrumentation.provider: newRelic
+    eventType: Span
+    relationship: CALLS
+    target:
+      domain: EXT
+      type: SERVICE
+      domainId: peer.hostname
+      encodeIdentifierInGUID: true
+    conditions:
+      - attribute: category
+        value: datastore
+
+  # SELECT count(*) From Span facet request.domain where request.domain is not null and
+  # entityGuid is not null
+  - instrumentation.provider: newRelic
+    eventType: Span
+    relationship: CALLS
+    target:
+      domain: EXT
+      type: SERVICE
```

Comment:
Is this rule actually wanted? AFAIK the entities that have the request.domain are those "external" pseudo entities that won't work until they are indexed.  If this is the case I'd remove this rule.

Reply:
All 4 of these rules need to be properly revised and likely just removed.
Saying that, I've manually searched through and found some existing open telemetry entities using that as the domainID. I think it's good enough data to test the pipeline with as we build it out.

Reply:
Yup, it would be interesting to know which type is being used in those cases (e.g. is it SERVICE or the THIRD_PARTY_SERVICE_ENTITY ?)

Reply:
we don't have any actual entities with `THIRD_PARTY_SERVICE_ENTITY` so it would have to be `SERVICE`

Reply:
Digging deeper though, none of these entities actually have telemetry right now. Something is not right.

Looking further still, all of the ext entities here are over a month old. Perhaps they are left over from when ext-service rules were too open

Reply:
Even though this rule will likely never actually match an existing target entity, it could still be handy for test. Ultimately, we don't want to generate or at least not show a relationship to a non existing entity. We can use this rule to test that case

Reply:
I'll leave a comment explaining